orchprovider/k8s/v1/util.go

```diff
@@ -457,7 +457,7 @@ type LabelK8sObject struct {
 	// to the targetted K8s Object
 	LabelValue string
 }
-
+// NewLabelK8sObject returns a LabelK8sObject
```

Comment:
change this to "returns an instance of LabelK8sObject whcih contains the key and value of the k8s labels"